diff -Nru axel-1.0a/conf.c axel-1.0a.new/conf.c
--- axel-1.0a/conf.c	Tue Feb 19 18:10:37 2002
+++ axel-1.0a.new/conf.c	Sun Mar 23 14:26:22 2003
@@ -148,6 +148,13 @@
 	if( !conf_loadfile( conf, ETCDIR "/axelrc" ) )
 		return( 0 );
 	
+	if( ( s2 = getenv( "ETC_DIR" ) ) != NULL )
+	{
+		sprintf( s, "%s/%s", s2, "axelrc" );
+		if( !conf_loadfile( conf, s ) )
+			return( 0 );
+	}
+	else
 	if( ( s2 = getenv( "HOME" ) ) != NULL )
 	{
 		sprintf( s, "%s/%s", s2, ".axelrc" );
